.github/workflows/ci.yml

```diff
@@ -107,7 +107,7 @@ jobs:
 
   docs-snippets:
     name: docs-snippets
-    if: github.event.pull_request.draft == false
+    if: (github.event.pull_request.draft == false && steps.changed-files.outputs.docs == 'true')
```

Comment:
This condition had not been migrated over yet

Reply:
It's actually what makes me a little nervous though. 

So in our current Azure pipeline, we run the same scope check that has now been added to the entire `docs-snippet` stage. https://github.com/great-expectations/great_expectations/blob/748fc89a21d8bfa7c29d40684be993037c217f49/ci/azure-pipelines-docs-integration.yml#L49 

This means the stage is only run for PRs where docs-related files (like `docs/**`) are changed. That is all fine and good. Here is my question:  Did this also mean that our release pipeline will not run these tests unless something has changed? Is that ok?

Reply:
Can you do an or here?

Run if docs-changed or event is not pull_request

Reply:
Great idea!